release-notes/cluster-observability-operator-release-notes.adoc

```diff
@@ -26,6 +26,101 @@ The following table provides information about which features are available depe
 
 include::snippets/unified-perspective-web-console.adoc[]
 
+
+[id="cluster-observability-operator-release-notes-1-3_{context}"]
+== {coo-full} 1.3
+
+The following advisory is available for {coo-full} 1.3:
+
+//* link:https://access.redhat.com/errata/RHBA-2025:XXXX[RHBA-2025:XXXX {coo-full} 1.3]
+
+[id="cluster-observability-operator-1-3-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+Incident detection feature reaches General Availability::
+The incident detection feature in the {coo-full} reaches General Availability (GA) status in this release for {ocp-product-title} versions 4.19+. Incident detection simplifies observability by grouping related alerts into incidents. This helps reduce alert fatigue and allows you to focus on the root cause of a problem. With General Availability (GA) status, the incident detection feature is production-ready and fully supported for enterprise deployments. Users can enable incident detection through the monitoring `UIPlugin` resource by setting `spec.monitoring.incidents.enabled: true`.
++
+For more information, see xref:../ui_plugins/monitoring-ui-plugin.adoc#coo-incident-detection-overview_monitoring-ui-plugin[Incident detection]
++
+link:https://issues.redhat.com/browse/COO-1281[COO-1281]
+
+
+Incidents view supports silenced alerts::
+The incidents view in the monitoring UI plugin supports silenced alerts in this release. Silenced alerts display with a `state` of `Silenced` in the incident detection UI and appear in a lighter color in the timeline. This helps you to distinguish them from active alerts and to understand the complete state of your cluster observability signals.
++
+link:https://issues.redhat.com/browse/COO-1280[COO-1280]
+
+Troubleshooting reaches General Availability::
+The troubleshooting feature in the {coo-full} reaches General Availability (GA) status in this release for {ocp-product-title} versions 4.19+. The troubleshooting UI plugin provides observability signal correlation, powered by the open source Korrel8r project. With the troubleshooting panel you can easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
++
+For more information, see xref:../ui_plugins/troubleshooting-ui-plugin.adoc#troubleshooting-ui-plugin[Troubleshooting UI plugin]
+
+End-to-end observability with ClusterObservability custom resource::
+The {coo-full} provides a `ClusterObservability` custom resource for deploying end-to-end observability with minimal configuration as a Technology Preview feature. The `ClusterObservability` custom resource simplifies the deployment of logging, tracing, and OpenTelemetry capabilities by automatically managing operator installation, storage configuration, and UI integration.
```

Comment:
The `ClusterObservability` custom resource should be changed as `observabilityinstallers`
@IshwarKanse please confirm